Packages/StatusKit/Sources/StatusKit/Row/Subviews/StatusRowCardView.swift

```diff
@@ -225,13 +225,14 @@ public struct StatusRowCardView: View {
 
 struct DefaultPreviewImage: View {
   @Environment(Theme.self) private var theme
+  @Environment(SceneDelegate.self) private var sceneDelegate
   
   let url: URL
   let originalWidth: CGFloat
   let originalHeight: CGFloat
 
   var body: some View {
-    _Layout(originalWidth: originalWidth, originalHeight: originalHeight) {
+    _Layout(originalWidth: originalWidth, originalHeight: min(originalHeight, sceneDelegate.windowHeight * 0.6)) {
```

Comment:
Need to think about this one, I don't think we should rely on scenelDelegate but we should provide an absolute height instead. Do you have a link to a problematic preview?

Reply:
https://mastodon.social/@hikingdude/111843640004810123 (on an iPad)